docs/style-guide.adoc

```diff
@@ -1,6 +1,14 @@
+:numbered:
```

Comment:
"Spacing" and "patterns" (paragraphs numbered 9 and 13) are both in bold, in contrast to all of the other headings. Either all should be bold (my preference) or none should be bold